### ISSUE TITLE
Help! How to use it in Laravel?

### ISSUE THREAD

**Issue by tsyrya** [eissasoubhi/summernote-bricks#1]
Please, provide an example of how to integrate it with Laravel. 
I've tried just compile it and put it in my js folder. But seems like the plugin looks for assets in a wrong directory. I've tried to set bricks_assets property, still doesn't work.

**Comment by eissasoubhi** [comment]
Can you provide me with the source code and the console error messages?

**Comment by tsyrya** [comment]
I found the reason. It was because of multiple instances of summernote on the page. It actually worked, but only with the last one.

**Comment by eissasoubhi** [comment]
yeah, it should be just one instance present in the page, do you want it to work with multiple instances?

**Comment by tsyrya** [comment]
yeah, that would be great :)

**Comment by nunodonato** [comment]
Hi! 
I was also trying to set this up in my laravel app, but I can't get it to work.
@eissasoubhi if it helps, this is the error messages I get in the console
![image](https://user-images.githubusercontent.com/1642015/127174819-26919591-f168-4aa3-9f69-fea207c022a1.png)

**Comment by eissasoubhi** [comment]
Closing as historical / no longer applicable to the v3 architecture. The original Laravel problem depended on the legacy `bricks_assets` loader and remote asset-directory layout. v3 removed that ownership model: Heading and Gallery are standalone Summernote plugins with normal built artifacts, while Bricks only composes already-registered toolbar buttons. Laravel (or any other framework) now only needs to serve/load the chosen jQuery + Summernote + plugin bundles through its normal asset pipeline. See the current README and `docs/V3_UPGRADE.md` for the v3 loading/migration contract.
